kura/org.eclipse.kura.core/OSGI-INF/metatype/org.eclipse.kura.data.DataService.xml

```diff
@@ -98,7 +98,36 @@
             default="0"
             min="0"
             description="Timeouts the in-flight messages congestion condition. The service will force a disconnect attempting to reconnect (0 to disable)."/>
-            
+
+        <AD id="average.publish.rate"
+            name="Average publish rate"
+            type="Integer"
+            cardinality="0"
+            required="true"
+            default="0"
+            min="0"
+            description="The average message publish rate. The unit of measurement is set by the publish rate unit of measurement option. If set to 0 the rate limit is disabled."/>
+        
+        <AD id="average.publish.rate.unit"
```

Comment:
Yes, but is it enough clear to the user set a rate of 0.1 message/second? If the maximum rate is 1 message/second, so we should change the unit to message/minute and get an integer value.